Source/WebCore/css/values/CSSValueConcepts.h

```diff
@@ -69,6 +69,17 @@ template<typename> inline constexpr auto TreatAsVariantLike = false;
 // The `VariantLike` concept can be used to filter to types that specialize `TreatAsVariantLike`.
 template<typename T> concept VariantLike = TreatAsVariantLike<T>;
 
+// Types can specialize this and set the value to true to be treated as "enum with
+// value representation" for CSS value type algorithms.
+// Requirements: Types must be an enum and specialize ValueRepresentation.
+// FIXME: This exists as a transitionary mechanism while not all enums specialize ValueRepresentation. Once they all do, this can be removed and std::is_enum_v<> can be used instead.
+template<typename> inline constexpr auto TreatAsEnumWithValueRepresentation = false;
```

Comment:
Should we do this in the opposite direction, with a default of true and we specialize right out of the gate for all the cases that need to be false? Presumably if we forget one we will get a compilation failure. I’d love to have a to do list that goes with the FIXME above.

Reply:
Good idea!!